akka-actor/src/main/scala/akka/serialization/Serializer.scala

```diff
@@ -351,6 +351,7 @@ final case class DisabledJavaSerializer(system: ExtendedActorSystem) extends Ser
   }
 
   override def fromBinary(buf: ByteBuffer, manifest: String): AnyRef = {
+    log.warning(LogMarker.Security, "Incoming message is using DisabledJavaSerializer resulting in Message class None! Manifest was: [{}]", manifest)
     this.fromBinary(empty, None)
```

Comment:
It was confusing to see the `None`, because it looked like the message that was tried to be deserialized was `None`.